docs/content/reference/routing-configuration/http/load-balancing/service.md

```diff
@@ -265,6 +265,12 @@ http:
 
 The mirroring is able to mirror requests sent to a service to other services. Please note that by default the whole request is buffered in memory while it is being mirrored. See the `maxBodySize` option in the example below for how to modify this behaviour. You can also omit the request body by setting the `mirrorBody` option to false.
 
+!!! warning "Default behavior of `percent`"
+
+    When configuring a `mirror` service, if the `percent` field is not set, it defaults to `0`, meaning **no traffic will be sent to the mirror**.
+    This can be unintuitive: if you define a mirror but forget the `percent`, it will be silently ignored.
+    Be sure to set the `percent` explicitly if you want mirroring to occur.
```

Comment:
In my opinion, we should remove those lines, as they essentially restate the first one.

```suggestion    
```